packages/api/openapi/licence.yaml

```diff
@@ -164,18 +164,24 @@ paths:
               schema:
                 $ref: '#/components/schemas/error'
 
-  '/user/{userId}/applications':
+  '/applications':
```

Comment:
Is this file auto-generated?

Reply:
No!

Reply:
Oh my life 😂 This must have been like 99% of the time taken haha!